ocrd_utils/ocrd_utils/constants.py

```diff
@@ -21,3 +22,11 @@
     '.jpeg': 'image/jpg',
     '.xml': MIMETYPE_PAGE
 }
+
+MIME_TO_EXT = {
+    'image/tiff': '.tif',
+    'image/png': '.png',
+    'image/jpg': '.jpg',
```

Comment:
It may also be `image/jpeg` (https://wiki.selfhtml.org/wiki/MIME-Type/%C3%9Cbersicht#I) !

Reply:
Added, thanks.